keyrings/live/blackbox-admins.txt

```diff
@@ -0,0 +1,5 @@
+alkazako@redhat.com
```

Comment:
The list of people with access to all access keys shouldn't be long :)

Any particular you see missing ?